material/partials/integrations/serviceworker.html

```diff
@@ -0,0 +1,10 @@
+{#-
+  This file was automatically generated - do not edit
+-#}
+<script>
+  if ("serviceWorker" in navigator) {
+    window.addEventListener("load", () => {
```

Comment:
This would need to go through babel, since its es6

Reply:
Oh, maybe? Note sure what the plans are for IE11 support? https://caniuse.com/#feat=arrow-functions

Reply:
Jep, this needs to be a proper `function`, not an arrow function.

Reply:
Done!